testbed/infrastructure.ts

```diff
@@ -0,0 +1,45 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as eks from '@aws-cdk/aws-eks';
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as cdk from '@aws-cdk/core'
+import { StackProps } from '@aws-cdk/core';
+
+export class Infrastructure extends cdk.Construct {
+    cluster: eks.Cluster;
+    constructor(scope: cdk.Construct, id: string, props: StackProps) {
+        super(scope, id);
+
+        const vpc = new ec2.Vpc(this, 'VPC', {});
+
+        this.cluster = new eks.Cluster(this, 'Cluster', {
+            vpc: vpc,
+            role: new iam.Role(this, 'ClusterRole', {
+                assumedBy: new iam.ServicePrincipal('eks.amazonaws.com'),
+                managedPolicies: [
+                    iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonEKSClusterPolicy'),
+                    iam.ManagedPolicy.fromAwsManagedPolicyName('AmazonEKSVPCResourceController')
+                ]
+            }),
+            version: eks.KubernetesVersion.V1_19,
```

Comment:
Thoughts on just letting this default?

Reply:
that hides the readability when someone wants to know in history for a given commit what the k8s version is ? We can trace it through library version though but not in our codebase. wdyt ?

Reply:
The problem is that this code will stop working in 6 months when EKS doesn't support v1.19 anymore.

Reply:
looks like version is mandatory per cdk lib